PeerioTechnologies/peerio-desktop
# chore: add new cert fingerprint

#### Relevant info and issue/PR links

https://app.clubhouse.io/peerio/story/17133/desktop-update-pinned-tls-certs-on

#### Testing instructions

We need staging or labs with the new cert deployed. Then check that Peerio can connect to it.

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
